tikv-client/src/main/java/com/pingcap/tikv/TiSession.java

```diff
@@ -205,10 +173,10 @@ public void close() throws Exception {
     getThreadPoolForTableScan().shutdownNow();
     getThreadPoolForIndexScan().shutdownNow();
     getPDClient().close();
+    channelFactory.close();
```

Comment:
I think ChannelFactory should be closed before PDClient.

Reply:
agreed.